### ISSUE TITLE
Disabled on mobile devices and mouse cursor style on hover

### ISSUE THREAD

**Issue by firmincol** [dshongphuc/magic-mouse-js#10]
Hi,
Thanks for your magic mouse.
It's awesome and really help me !
However, I have 2 issues I can't fix:

-  I can't disable it on tablet and mobile devices ! I pasted your CDN js and added directly your css on my server. Everything works great but I can't find a way to disable it on mobile.

- I'd like to have the mouse cursor to be different on hover (size and color). I tried different css lines but without result.

I really appreicate if you can give me a solution for this.

**Comment by dshongphuc** [comment]
Hi @firmincol , I checked the CDN code and it seem everything is good. Can you post your website here for me to check?

**Comment by firmincol** [comment]
No problem : www.lecolletgourmand.be

I tried again on mobile and the magic mouse is still there.

Thanks for your help

**Comment by habitualshaker** [comment]
It looks like Magic Mouse will appear on all devices over 768px. Would be better if possible to incorporate a check to see if a mouse is being used and only activate it then?

**Comment by dshongphuc** [comment]
Hi @habitualshaker  & @firmincol , sorry for my delay !
I was thinking about using 3nd party library to detect mobile & tablet device long time ago, but at that time I see other libs is too much heavily just for this purpose. I really want Magicmouse.js to be super lightweight, so I don't want to use 3nd libs.
Now I excited to let you know that I've found another lightweight libs to do exactly what we want to, it's just detect if device is touchable, it's [Modernizr](https://github.com/modernizr/modernizr) (it support a lot of features but I only get the "touch screen event"). 

I've updated the source code and CDN, this issue has been solved in version 1.1 : https://res.cloudinary.com/veseylab/raw/upload/v1613706377/magicmouse/magic_mouse-1.1.js

Thank you ! Hope you enjoy it, let me know if any problem with this.
Notice : I have not update the live demo to latest version, will do it later when have time
